docs/content/tutorial/building-an-asset-graph.mdx

```diff
@@ -26,18 +26,24 @@ Copy and paste the completed asset code below into `assets.py`:
   <code>assets.py</code>
 </Note>
 
-```python
-import pandas as pd # Add new imports to the top of `assets.py`
+```python file=/tutorial/building_an_asset_graph/assets_initial_state.py startafter=start_topstories_asset endbefore=end_topstories_asset
+import pandas as pd  # Add new imports to the top of `assets.py`
+import requests
+
+from dagster import asset
+
 
 @asset
-def topstories(topstory_ids): #this asset is dependent on topstory_ids
+def topstories(topstory_ids):  # this asset is dependent on topstory_ids
     results = []
     for item_id in topstory_ids:
-        item = requests.get(f"https://hacker-news.firebaseio.com/v0/item/{item_id}.json").json()
+        item = requests.get(
+            f"https://hacker-news.firebaseio.com/v0/item/{item_id}.json"
+        ).json()
         results.append(item)
 
         if len(results) % 20 == 0:
-            print(f"Got {len(results)} items so far.")
+            print(f"Got {len(results)} items so far.") # noqa: T201
```

Comment:
❓ What's this comment for?

Reply:
Oh yeah, I wanted to ask about this.

`ruff` is reasonably complaining about the print statement, but that print statement needs to stay, so that comment ignores checking for rule T201 (no prints).

I did a quick check and wasn't sure if I should've done it inline or extended `doc_snippets/pyproject.toml`.

but now looking at it...I probs should've just done that.